jobs/config.json

```diff
@@ -11625,7 +11625,7 @@
       "make",
       "verify",
       "WHAT=typecheck",
-      "KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/artifacts"
+      "KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/_artifacts"
```

Comment:
whoops yeah. this is kinda dumb, I don't know why we do _artifacts locally and /artifacts on the remote :/